podcastparser.py

```diff
@@ -868,6 +889,18 @@ def add_itunes_owner(self):
     def append_itunes_owner(self, key, value):
         self.data['itunes_owner'][key] = value
 
+    def add_itunes_categories(self):
+        self.data['itunes_categories'] = []
+
+    def append_itunes_category(self, key, value):
+        if self.data.get('itunes_categories', None) is None:
+            self.add_itunes_categories()
+        self.data['itunes_categories'].append([value])
```

Comment:
This can be simplified using [`dict.setdefault()`](https://docs.python.org/3/library/stdtypes.html#dict.setdefault) to:
```python3
     def append_itunes_category(self, key, value):
         self.data.setdefault('itunes_categories', []).append([value])
```

Reply:
Nice!